toolchain/sem_ir/ids.h

```diff
@@ -237,9 +237,13 @@ struct InstBlockId : public IdBase, public Printable<InstBlockId> {
   using ElementType = InstId;
   using ValueType = llvm::MutableArrayRef<ElementType>;
 
-  // All File instances must provide the 0th instruction block as empty.
+  // An empty block, reused to avoid allocating empty vectors. Always the
+  // 0-index block.
   static const InstBlockId Empty;
 
+  // Exported instructions. Always the 1-index block.
```

Comment:
I think it'd be useful to mention here that this block is empty until the file has been fully checked.

Reply:
Done